includes/server/class-llms-rest-access-plans-controller.php

```diff
@@ -636,7 +637,7 @@ private function add_subordinate_props( &$to_set, $saved_props, $creating ) {
 		} else {
 
 			$subordinate_props['is_free']     = 'yes';
-			$subordinate_props['price']       = 0;
+			$subordinate_props['price']       = (float) 0;
```

Comment:
An alternative to casting. Either way works though.
```suggestion
			$subordinate_props['price']       = 0.0;
```

Reply:
yeah I guess your way is better <3